great_expectations/core/batch.py

```diff
@@ -982,12 +1141,45 @@ def get_batch_request_from_acceptable_arguments(  # noqa: C901 - complexity 21
 
         batch_spec_passthrough
 
+        batch_request_options
+
         **kwargs
 
     Returns:
-        (BatchRequest or RuntimeBatchRequest) The formal BatchRequest or RuntimeBatchRequest object
+        (BatchRequest, RuntimeBatchRequest or FluentBatchRequest) The formal BatchRequest, RuntimeBatchRequest or FluentBatchRequest object
     """
 
+    # block-style batch-request args, includes arguments for both runtime and basic batch requests
+    block_config_args = {
+        "batch_data": batch_data,
+        "query": query,
+        "path": path,
+        "runtime_parameters": runtime_parameters,
+        "data_connector_query": data_connector_query,
+        "batch_identifiers": batch_identifiers,
+        "batch_filter_parameters": batch_filter_parameters,
+        "limit": limit,
+        "index": index,
+        "custom_filter_function": custom_filter_function,
+        "sampling_method": sampling_method,
+        "sampling_kwargs": sampling_kwargs,
+        "splitter_method": splitter_method,
+        "splitter_kwargs": splitter_kwargs,
+        "batch_spec_passthrough": batch_spec_passthrough,
+    }
+
+    def block_style_args() -> list[str]:
```

Comment:
I use this to help with parameter checking if `data_connector_name` is missing; open to ideas on how to make it better (and the dict declaration above)